src/ProgressCircle.js

```diff
@@ -0,0 +1,64 @@
+import "./ProgressBar.scss";
+
+function ProgressCircle(props) {

```

Comment:
Same deal as bus stop: I think it's more readable/usable when you unpack the props